ci/requirements_optional.txt

```diff
@@ -1,7 +1,2 @@
 grad-cam==1.4.6
```

Comment:
General idea for optional: **let's remove it.**

1. Let's remove jupyer because we don't use it anymore. We had this requirement to test a notebook that has already been removed from the library.

2. After that we have the only optional req -- gradcam. I see two options. First, we can move it into `cv` tag. The problem here is gradcam has some extra [dependencies](https://github.com/jacobgil/pytorch-grad-cam/blob/master/requirements.txt), but at the same time it's value is relatively small and I don't think all `cv` users really not to deal with installing gradcam. Thus, I suggest the second option: move gradcam to the `tests`. So, the users will install this library in those rare cases where they need it (similar to loggers logic).

What do u think?

Reply:
1 - agreed. 
2 - IMHO gradcam not for test. it is for cv. And yes - it has a lot of dependencies, especially scikit-learn. Lets move it to test, maybe later will be better solution.

Reply:
2 - Agree with you, @ayasyrev . Yep, we need loggers and gradcam in tests, but it's file naming is a bit weir. We may have `requirements_optional.txt`, where we put gradcam and loggers, but we don't make `optional` tag for cv. Instead, we just install this requirements in test time. As for the `requirements_test.txt`, we just keep there pytest and linters, which is more semantically correct in terms of naming. What do u think?

Reply:
OK, test - for test related stuff.